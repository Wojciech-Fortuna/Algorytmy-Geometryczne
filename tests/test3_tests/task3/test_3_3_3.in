0.002417337356075161 0.04811376394010061
-0.006453630385860332 0.03126327374402216
-0.024417340063279683 0.021152979626375107
-0.006675404579408711 0.009020626685198624
-0.034175404579408715 -0.0024377066481347104
-0.006231856192311938 -0.013559030177546477
-0.045929436837473234 -0.029735500765781776
-0.006453630385860332 -0.036138687040291585
-0.004679436837473225 -0.04422692233440923
-2.2178772957098802e-05 -0.0499560890010759
0.0037479825173654807 -0.04523795174617394
0.004856853485107421 -0.03479064782460531
0.04322378896897838 -0.026365402726566088
0.0050786276786558004 -0.012548000765781775
0.02548185348510741 -0.0010896674324484407
0.005522176065752574 0.010368665900884907
0.01616733735607516 0.02317503844990451
0.0050786276786558004 0.034633371783237846