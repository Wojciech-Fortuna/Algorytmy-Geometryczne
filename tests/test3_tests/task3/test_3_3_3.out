1 17
16 1
16 2
2 15
3 15
3 14
4 14
13 4
5 13
12 5
6 12
11 6
7 11
8 11
10 8